_SpecRunner.html

```diff
@@ -3,7 +3,7 @@
 <head>
   <meta charset="utf-8">
   <title>Jasmine Spec Runner</title>
-  <link rel="shortcut icon" type="image/png" href=".grunt/grunt-contrib-jasmine/jasmine_favicon.png">
```

Comment:
This is a windows vs macOS thing, I believe. Can we just git ignore this file? Otherwise one of us is going to have it as dirty all of the time. Or perhaps an upgrade to the grunt jasmine plugin would fix it? I'm not sure that this happens on our newer projects.

Reply:
No clue. I don't think any on my packages have upgrades. Happy to ignore the spec file though

Reply:
I vote to ignore it.

Reply:
ignored